templates/welcome.html

```diff
@@ -104,26 +83,34 @@
 <div class="container-fluid">
     <div class="row">
         <div class="col-md-3"></div>
-        <div class="col-md-5">
-            <h2 class="welcome">Welcome <span id="sessionname"></span>!</h2>
+        <div class="col-md-6">
+            <h2 class="welcome">Welcome!</h2>
```

Comment:
slightly out of line:
<img width="294" alt="screen shot 2018-09-06 at 5 49 14 pm" src="https://user-images.githubusercontent.com/2499235/45169198-5ecaee80-b1fd-11e8-8a2f-266cf749422a.png">